app/domain/authentication/authn_jwt/orchestrate_authentication.rb

```diff
@@ -0,0 +1,47 @@
+require 'command_class'
+
+# This class is the starting point of the JWT authenticate requests, responsible to identify the vendor configuration and to run the JWT authenticator
+module Authentication
+  module AuthnJwt
+
+    OrchestrateAuthentication ||= CommandClass.new(
+      dependencies: {
+        validate_uri_based_parameters: Authentication::AuthnJwt::InputValidation::ValidateUriBasedParameters.new,
+        configuration_factory: Authentication::AuthnJwt::VendorConfigurations::ConfigurationFactory.new,
+        jwt_authenticator: Authentication::AuthnJwt::Authenticator.new,
+        logger: Rails.logger,
+        enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str
+      },
+      inputs: %i[authenticator_input]
+    ) do
+      def call
+        validate_uri_based_parameters
+        authenticate_jwt
+      end
+
+      private
+
+      def authenticate_jwt
+        relevant_authenticator = authenticator_name
+        @logger.info(LogMessages::Authentication::AuthnJwt::JwtAuthenticatorEntryPoint.new(relevant_authenticator))
```

Comment:
can't we do this?
```suggestion
        @logger.info(LogMessages::Authentication::AuthnJwt::JwtAuthenticatorEntryPoint.new(authenticator_name))
```

Reply:
Done this 
`def authenticate_jwt
        @logger.info(LogMessages::Authentication::AuthnJwt::JwtAuthenticatorEntryPoint.new(relevant_authenticator))

        jwt_authenticator_configuration = @configuration_factory.call(
          authenticator_input: @authenticator_input
        )
        @jwt_authenticator.call(
          jwt_configuration: jwt_authenticator_configuration,
          authenticator_input: @authenticator_input
        )
      end

      def relevant_authenticator
        @relevant_authenticator ||= @authenticator_input.authenticator_name
      end`

Reply:
but why do we need the variable `relevant_authenticator`? why is it better than `authenticator_name`? we already have a variable, let's use it.